stdlib/LinearAlgebra/test/addmul.jl

```diff
@@ -189,7 +190,8 @@ end
 
                     returned_mat = mul!(C, Af, Bf, α, β)
                     @test returned_mat === C
-                    @test collect(returned_mat) ≈ α * Ac * Bc + β * Cc  rtol=rtol
+                    # This test is skipped because it is flakey, but should be fixed and put back (see #49966)
```

Comment:
Ah!  I thought I broke it in https://github.com/JuliaLang/julia/pull/49405#issuecomment-1567183600

Reply:
haha, I don't think so